game-app/game-headed/src/main/java/games/strategy/engine/lobby/client/ui/LobbyGamePanel.java

```diff
@@ -190,7 +192,16 @@ private void joinGame() {
     // we sort the table, so get the correct index
     final GameDescription description =
         gameTableModel.get(gameTable.convertRowIndexToModel(selectedIndex));
-    GameProcess.joinGame(description, lobbyClient.getUserName());
+    final GameDescription.GameStatus status = description.getStatus();
+    if (GameDescription.GameStatus.LAUNCHING == status) {
```

Comment:
I just wanted to know, this return could go on this same line of the if without opening the braces

Reply:
Yes, it could. This is a style choice.  This project wishes for non-mandatory braces to always be used for if statements.  

Preferred code style is documented here: https://github.com/triplea-game/triplea/blob/master/docs/development/reference/code-conventions/java-code-conventions.md

Which largely says to follow google java style guide, which is what says to use the optional curly braces:
https://google.github.io/styleguide/javaguide.html#s4.1.1-braces-always-used

There is an auto-formatter as well to remove a lot of the other style decisions (which can be run via a gradle plugin 'spotless', and google-java-format plugin can be installed into IDE to run it) ; hence the style documentation intentionally does not spell out some of the low level formatting that is needed and is otherwise enforced by 'spotless'